docs/UsersGuide/source/AQM.rst

```diff
@@ -170,3 +132,128 @@ Users may check experiment status from the experiment directory with either of t
    # Check the experiment status and relaunch the workflow (for manual jobs)
    ./launch_FV3LAM_wflow.sh; tail -n 40 log.launch_FV3LAM_wflow
 
+
+WE2E Test for AQM
+=======================
+
+Build the app for AQM:
+
+.. code-block:: console
+
+  ./devbuild.sh -p=hera -a=ATMAQ
+
+
+Add the WE2E test for AQM to the list file:
+
+.. code-block:: console
+
+   echo "custom_ESGgrid" > my_tests.txt
+   echo "aqm_grid_AQM_NA13km_suite_GFS_v16" >> my_tests.txt
+
+
+Run the WE2E test:
+
+.. code-block:: console
+
+   $ ./run_WE2E_tests.py -t my_tests.txt -m hera -a gsd-fv3 -q
+
+
+
+Additional Tasks for AQM
+===============================
+
+Structure of SRW-AQM
+-------------------------
+
+The flowchart of the non-DA (data assimilation) SRW-AQM (Air Quality Modeling) is illustrated in :numref:`Figure %s <FlowProcAQM>`. Compared to the non-coupled (ATM stand-alone) FV3-LAM, SRW-AQM has additional tasks for pre- and post-processing. For pre-processing, multiple emission data such as NEXUS, fire, and point-source emission are retrieved or created for air quality modeling. Moreover, the chemical initial conditions (ICs) are extracted from the restart files of the previous cycle and added to the existing IC files. The chemical lateral boundary conditions (LBCs) and the GEFS aerosol data are also adeded to the existing LBC files. For post-processing, air quality forecast products for O3 and PM2.5 are generated and the bias-correction technique is applied to improve the accuracy of the results.
+
+.. _FlowProcAQM:
+
+.. figure:: https://github.com/ufs-community/ufs-srweather-app/wiki/SRW-AQM_workflow.png
+      :alt: Flowchart of the SRW-AQM tasks.
```

Comment:
The figure isn't coming through in the built rtd document.  I was able to correct this by aligning line 173 with 172:

```suggestion
.. figure:: https://github.com/ufs-community/ufs-srweather-app/wiki/SRW-AQM_workflow.png
   :alt: Flowchart of the SRW-AQM tasks.
```

Reply:
@MichaelLueken, thanks for your help!!! Modified.

Reply:
Thanks, @chan-hoo!  The figure is now appearing in the documentation!  Resolving now.